src/pages/CRLP_RollupSetup.page

```diff
@@ -0,0 +1,26 @@
+<!--
```

Comment:
Removed the IntelliJ comment, but in looking quickly for another header on another page to copy, do any of our other pages use the standard copyright header? Or is this another item for the backlog?

Reply:
Currently, none of our VF or Components have a copyright, but I raised the question with Kevin and he noted that it was something that they discussed, but never quite implemented. Since these are being built from scratch, we should include the copyright header from the start. You can copy a copyright header from an apex class.